cms/project_template/project_name/static/js/ckeditor_styles.js

```diff
@@ -0,0 +1,6 @@
+CKEDITOR.stylesSet.add('custom_styles', [
+    // Block-level styles
+    {name: 'Button', element: 'a', attributes: {'class': 'button radius'}},
```

Comment:
Given the changes made in #68, it might be worth using the new `c-button` (or whatever the class is now) rather than this one.